### PR TITLE
ICMSLST-391 OIL Application - Submit

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

File: web/domains/case/_import/forms.py [hunk 1]
```diff
@@ -190,3 +190,18 @@ class Meta:
...
             "dealer": "Did you buy from a dealer?",
         }
+
+
+class SubmitOILForm(forms.Form):
+    confirmation = forms.CharField(
+        label='Confirm that you agree to the above by typing "I AGREE", in capitals, in this box'
+    )
+
+    def clean(self):
```
Comment (oskusalerma) [inline web/domains/case/_import/forms.py:200]: is there a reason you're not implementing this in `clean_confirmation`? less code, more standard for one-field checks

Comment (kevinetienne) [reply web/domains/case/_import/forms.py]: Changed in https://github.com/uktrade/icms/pull/251/commits/d7955586a9566a040ebef064fdbde2655a42f04d

File: web/domains/case/_import/forms.py [hunk 2]
```diff
@@ -190,3 +190,19 @@ class Meta:
...
+class SubmitOILForm(forms.Form):
+    confirmation = forms.CharField(
+        label='Confirm that you agree to the above by typing "I AGREE", in capitals, in this box'
+    )
+
+    def clean_confirmation(self):
+        confirmation = self.cleaned_data["confirmation"]
+
+        if confirmation != "I AGREE":
+            self.add_error(
```
Comment (oskusalerma) [inline web/domains/case/_import/forms.py:204]: `clean_XXX` methods should raise ValidationError

**Review by oskusalerma — CHANGES_REQUESTED** [review]

File: web/domains/case/_import/forms.py (hunk 2 above)
Comment (kevinetienne) [reply web/domains/case/_import/forms.py:204]: thanks

Comment (kevinetienne) [reply web/domains/case/_import/forms.py]: Fixed in https://github.com/uktrade/icms/pull/251/commits/003e7f6c1a9c213b4b52dc6d712a07bb63ff842a

**Review by oskusalerma — APPROVED** [review]

**Pull request merged**